### PR TITLE
ansible: fix etcd server list

### PR DESCRIPTION
**Pull request by safchain**
now generated from the analyzers list

### CODE REVIEW

File: contrib/ansible/inventory/hosts.localhost [hunk 1]
```diff
@@ -4,6 +4,7 @@ agents
 
 [skydive:vars]
 skydive_listen_ip=0.0.0.0
+skydive_etcd_port=12345
```
Comment (nplanel) [inline contrib/ansible/inventory/hosts.localhost]: 12379 ?

Comment (safchain) [reply contrib/ansible/inventory/hosts.localhost]: that an example, We can put anything. Anyway I will remove this line as it was to do my own tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by masco-enovance** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by nplanel** [comment]
could you cherry-pick https://github.com/nplanel/skydive/commit/bdb8309c912c2dd919d13bc78da88d9603a38692

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
rerun scale-tests

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
rerun scale-tests

**Comment by safchain** [comment]
rerun scale-tests

**Pull request merged**